### PR TITLE
Makes the nuke beerkeg easily spawnable.

### PR DESCRIPTION
**Pull request by MarsM0nd**
Moves most of the edits on the beerkeg that is on metastation to it's own subtype of beerkegs, for ease of access to it. Also changes the metastation beerkeg to that subtype.
Regarding the map edit, i manually removed the edits and changed the type of the beerkeg, instead of using mapping tools. Tested, and didn't seem to break anything, if such manual edits do break anything i'm not aware off, please say so.
No changelog as there is no difference for players, only makes the beerkeg easily spawnable incase someone needs to nuke their liver with it.

### CODE REVIEW

**Comment by KingPhilipIII** [comment]
Speed merge so I can pray for nuke beerkegs.

File: code/modules/reagents/reagent_dispenser.dm [hunk 1]
```diff
@@ -253,6 +253,14 @@
 	explosion(loc, 0, 3, 5, 7, 10)
 	qdel(src)
 
+/obj/structure/reagent_dispensers/beerkeg/nuke
+	name = "NanoTrasen-brand nuclear fission explosive"
```
Comment (SamHPurp) [inline code/modules/reagents/reagent_dispenser.dm]: `Nanotrasen`, not `NanoTrasen`.

File: code/modules/reagents/reagent_dispenser.dm [hunk 2]
```diff
@@ -253,6 +253,14 @@
 	explosion(loc, 0, 3, 5, 7, 10)
 	qdel(src)
 
+/obj/structure/reagent_dispensers/beerkeg/nuke
+	name = "NanoTrasen-brand nuclear fission explosive"
+	desc = "One of the more successful achievements of the NanoTrasen Corporate Warfare Division, their nuclear fission explosives are renowned for being cheap\
```
Comment (SamHPurp) [inline code/modules/reagents/reagent_dispenser.dm]: `Nanotrasen`, not `NanoTrasen`.

File: code/modules/reagents/reagent_dispenser.dm [hunk 3]
```diff
@@ -253,6 +253,14 @@
 	explosion(loc, 0, 3, 5, 7, 10)
 	qdel(src)
 
+/obj/structure/reagent_dispensers/beerkeg/nuke
+	name = "NanoTrasen-brand nuclear fission explosive"
+	desc = "One of the more successful achievements of the NanoTrasen Corporate Warfare Division, their nuclear fission explosives are renowned for being cheap\
+	to produce and devestatingly effective. Signs explain that though this is just a model, every NanoTrasen station is equipped with one, just in case. \
```
Comment (SamHPurp) [inline code/modules/reagents/reagent_dispenser.dm]: `Nanotrasen`, not `NanoTrasen`.

**Comment by MarsM0nd** [comment]
Did change to Nanotrasen, did only copy it from the metastation edits, meta does have a few NanoTrasens instead of Nanotrasen it seems, but changing that is not within the scope of this PR. Also a bit too quick with the comitting, no idea how a Nanotrasen did land there...

**Pull request merged**